src/Xray/XrayScanClient.ts

```diff
@@ -84,15 +88,23 @@ export class XrayScanClient {
      * @param watches - List of Watches or undefined
      * @returns URL for "POST api/v1/scan/graph"
      */
-    private getUrl(projectKey: string | undefined, watches?: string[]): string {
+    private getUrl(projectKey: string | undefined, watches?: string[], multiScanId?: string, technologies?: string[]): string {
         let url: string = XrayScanClient.scanGraphEndpoint;
+        let params: string[] = [];
+
         if (projectKey && projectKey.length > 0) {
-            url += `?project=${projectKey}`;
+            params.push(`project=${projectKey}`);
         } else if (watches && watches.length > 0) {
-            url += '?watch=' + watches.join('&watch=');
```

Comment:
looks like a positive potential bug fix here. Do we have a test that watches were sent with projects beforehand? 
anyway - now that we have 4 - need to make sure several permutations are covered here.

Reply:
yes, we do have tests for that, at the matching `.spec` file